spec/features/anonymous_user_happy_spec.rb

```diff
@@ -22,8 +34,28 @@
   end
 
   scenario 'they expected the most popular links' do
+    url_1 = Url.create!(original: 'https://www.twitter.com', popularity: 8)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.